samples/word/Basics/basic-doc-assembly.yaml

```diff
@@ -0,0 +1,241 @@
+id: ciyvyru7l00014r6ydkzkwka7

```

Comment:
I see a ^M at the end of each line in the first section of the file. Not sure how that got there.